src/input_parsers/OnnxParser.h

```diff
@@ -0,0 +1,91 @@
+/*********************                                                        */
+/*! \file OnnxParser.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Matthew Daggitt
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** A parser for ONNX neural network files.
+ **/
+
+#ifndef __OnnxParser_h__
+#define __OnnxParser_h__
+
+#include "onnx.proto3.pb.h"
+#include "Map.h"
+#include "List.h"
+#include "Vector.h"
+#include "Set.h"
+#include "MString.h"
+#include "InputQuery.h"
+#include "NetworkParser.h"
+
+class OnnxParser : public NetworkParser
+{
+public:
+    OnnxParser( const String& path );
+    void generateQuery( InputQuery &inputQuery );
+    void generatePartialQuery( InputQuery &inputQuery , Set<String> &inputNames, String &outputName );
+
+private:
+
+    onnx::GraphProto _network;
+    Set<String> _inputNames;
+    String _outputName;
+    Set<String> _processedNodes;
+    uint _numberOfFoundInputs;
```

Comment:
The type here should be unsigned, since uint is not a standard type and its support is compiler dependent.

Reply:
Fixed